.github/workflows/coverall.yml

```diff
@@ -0,0 +1,20 @@
+on: ["push", "pull_request"]
+name: Test Coveralls
```

Comment:
maybe better name so it is clear in check section what is done, maybe `Check test coverage`?